### PR TITLE
updating ato text on index

### PR DESCRIPTION
**Pull request by leahbannon**

### CODE REVIEW

File: _pages/index.md [hunk 1]
```diff
@@ -15,19 +15,19 @@ The goal of this site is to provide 18F staff with the information they need to
 
 ### Compliance
 
-* You need an Authority to Operate (ATO). At a very high level, it covers the security reviews and approvals **required** for public government websites. It is very important and can take months (albeit rarely). [Learn about requirements and timelines for getting your ATO here.](ato/)
+* You need an Authority to Operate (ATO). Essentially, ATOs cover the important security reviews and approvals **required** for public government websites. They can take months (albeit rarely). You should review this [overview of ATOs](ato/) and these [tips](/ato/tips/) to start planning.
```
Comment (afeld) [inline _pages/index.md]: Mind removing the leading slash on `/ato/tips/` to fix the link?

File: _pages/index.md [hunk 2]
```diff
@@ -39,14 +39,15 @@ The goal of this site is to provide 18F staff with the information they need to
...
+* Is your project accessible and [Section 508](laws/508/) compliant? The team will need to incorporate this throughout the project, but you'll also need to set up a review at least 2 weeks before launch.
+* How good is your code test coverage? Before shipping, you should have codecov badges on your GitHub repo READMEs and coverage should be above 90% (green). (This is not a perfect measure for code quality, but a helpful check.) The testing working group recommends reviewing your status early and often. _Ask the #wg-testing if you have questions._
+* Are your APIs up to [18F's API Standards](https://github.com/18f/api-standards)? _Ask the #wg-api if you have questions._
+* Have you installed Google Analytics and [DAP](https://www.digitalgov.gov/services/dap/) on everything? You can learn more about how to do this in our [Analytics Standards](https://github.com/18F/analytics-standards). _Ask the #wg-analytics if you have questions._
```
Comment (afeld) [inline _pages/index.md]: Maybe remove the "the" before the channel names for these three?

**Comment by afeld** [comment]
Looks good! Just a couple minor things.

File: _pages/index.md (hunk 2 above)
Comment (leahbannon) [inline _pages/index.md]: good call

**Comment by leahbannon** [comment]
done, thanks!

**Pull request merged**